src/theme/themeStyles.ts

```diff
@@ -6,18 +6,93 @@ const materialTheme = createTheme({
   palette: {
     primary: {
       main: theme.primaryColor,
+      contrastText: theme.light.light,
+      light: theme.light.tabBackgroundColor,
+    },
+    background: {
+      base: theme.light.backgroundBase,
+      default: theme.light.light,
+    },
+    error: {
+      main: theme.light.dangerColor,
+    },
+    text: {
+      primary: theme.light.primaryFontColor,
+      secondary: 'rgba(0, 0, 0, 0.6)',
+    },
+  },
+  typography: {
+    fontFamily: [
+      '-apple-system',
+      'BlinkMacSystemFont',
+      '"Segoe UI"',
+      'Roboto',
+      '"Helvetica"',
+      'Arial',
+      'sans-serif',
+      '"Apple Color Emoji"',
+      '"Segoe UI Emoji"',
+      '"Segoe UI Symbol"',
+    ].join(','),
+    h1: {
+      fontSize: theme.fontSizes.fontRemLarge,
+    },
+    h2: {
+      fontSize: theme.fontSizes.fontRemMedium,
+      fontWeight: theme.fontWeight,
+    },
+    button: {
+      fontSize: theme.fontSizes.fontRemSmall,
+      lineHeight: 1.3333,
     },
   },
   components: {
     MuiCssBaseline: {
       styleOverrides: {
         body: {
-          fontSize: '0.875rem',
+          fontSize: theme.fontSizes.fontRemXSmall,
           lineHeight: 1.43,
           letterSpacing: '0.01071em',
         },
       },
     },
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          borderRadius: 28,
+          textTransform: 'capitalize',
+          padding: '12px 30px',
+        },
+        containedSizeSmall: {
+          padding: '5px 10px',
+          borderRadius: 14,
+        },
+        text: {
+          padding: 0,
+          borderRadius: 0,
+          fontSize: 'inherit',
+          fontWeight: theme.fontWeight,
+        },
```

Comment:
There seems to be a change in the styling of text buttons seen in Recurring Donations on clicking Edit Donation and viewing the calendar picker. This was also seen for Treemapper>Import data

*Before*
![Screenshot 2022-12-16 at 11 47 27 AM](https://user-images.githubusercontent.com/44917347/208036889-fd5ae3c1-61f8-49a6-8868-df29dad472af.png)

*After*
![Screenshot 2022-12-16 at 11 47 31 AM](https://user-images.githubusercontent.com/44917347/208036885-2ae3acf4-677f-41cc-869f-9024cf542c53.png)

Reply:
@prachigarg19 This likely comes from the button style overrides added to themeStyle.ts - inspect the elements and try to figure out where the styles are coming from.

```
MuiButton: {
      styleOverrides: {
        root: {
          borderRadius: 28,
          textTransform: 'capitalize',
          padding: '12px 30px',
        },
        containedSizeSmall: {
          padding: '5px 10px',
          borderRadius: 14,
        },
        text: {
          padding: 0,
          borderRadius: 0,
          fontSize: 'inherit',
          fontWeight: theme.fontWeight,
        },
        
```

Reply:
This is done for the EditModal, but is pending for other places where MuiDatePicker is used.